docs/medical-api/more-info/webhooks.mdx

```diff
@@ -284,3 +282,52 @@ The format follows:
 
   </Expandable>
 </ResponseField>
+
+#### Medical Record
+
+When querying for a Patient's consolidated data, you can also request this information as a pdf or html.
+The following is the result of the [previous section](#patient-consolidated-data), but with the `Bundle` containing a document reference to the medical record that was converted from the requested FHIR bundle.
```

Comment:
nit: "to a rendered medical record that was represents the resulting FHIR bundle."